OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -70,88 +77,145 @@ public void InvokeCommand(string name, string arg)
 				return;
 			}
 
-			switch (name)
+			if (!commandHandlers.TryGetValue(name, out var command))
+				return;
+
+			command.Item2(arg, world);
+		}
+
+		static bool GiveCash(string arg, World world)
+		{
+			IssueCashDevCommand(world, "DevGiveCash", arg);
+			return true;
```

Comment:
The return value isn't used, so declare these all as void and remove the `returns` to save a bunch of unnecessary lines.

Reply:
That won't work. The `Func` requires a return value and doesn't accept `void`.